### PR TITLE
This small fix should enable a successful import of MS_functions afte…

### PR DESCRIPTION
**Pull request by HannoSpreeuw**
…r 'conda install nb_conda'

### CODE REVIEW

File: matchms_01_MGF_from_gnps_library_search.ipynb [hunk 1]
```diff
@@ -39,8 +54,9 @@
     "from config import ROOT, PATH_SPEC2VEC, PATH_MS_DATA, PATH_SAVE_MODEL, PATH_COMPUTED, PATH_OUTPUT\n",
     "\n",
     "sys.path.insert(0, PATH_SPEC2VEC)\n",
+    "print(PATH_SPEC2VEC)\n",
     "\n",
-    "import MS_functions #import Spec2Vec functions related to MS spectra"
+    "from matchms import MS_functions #import Spec2Vec functions related to MS spectra"
```
Comment (jspaaks) [inline matchms_01_MGF_from_gnps_library_search.ipynb:59]: I think you meant to commit only this line, right?

**Review by jspaaks — CHANGES_REQUESTED** [review]
I also would have expected the ``nb_conda`` to show up somewhere?

**Comment by jspaaks** [comment]
BTW I unassigned Faruk because we agreed not to push work onto others' todo list --they can pull if they want. To that end I've added this PR to the sprint board as well.

File: matchms_01_MGF_from_gnps_library_search.ipynb (hunk 1 above)
Comment (HannoSpreeuw) [reply matchms_01_MGF_from_gnps_library_search.ipynb:59]: Yes

Comment (HannoSpreeuw) [reply matchms_01_MGF_from_gnps_library_search.ipynb:59]: The nb_conda requirement can perhaps be added to environment.yml in matchms?

Comment (jspaaks) [reply matchms_01_MGF_from_gnps_library_search.ipynb:59]: (Be aware I'm not very familiar with notebooks or conda, but...)
...actually, I would expect ``nb_conda`` in ``notebooks``'s ``environment.yml``, not ``matchms``'s. And also, we should add whatever is imported in the notebooks, e.g. ``matplotlib``,  ``numpy`` and such (only what is used directly, not dependencies of dependencies). After we release ``matchms`` on ``conda``  and/or ``pip`` I think we can add it to ``environment.yml`` as well.

Comment (HannoSpreeuw) [reply matchms_01_MGF_from_gnps_library_search.ipynb:59]: Your are right, `nb_conda` should be added to an environment.yml in the notebooks repo. That file does not exist yet.

Comment (HannoSpreeuw) [reply matchms_01_MGF_from_gnps_library_search.ipynb:59]: I also agree with your second comment in the sense that the environment.yml of the notebooks repo should include these dependencies as well.

```
After we release matchms on conda and/or pip I think we can add it to environment.yml as well.
```
What do you mean by `it`?

Comment (jspaaks) [reply matchms_01_MGF_from_gnps_library_search.ipynb:59]: My suggestion was to add ``it``=``matchms`` to the to-be-created ``environment.yml`` in ``matchms/notebooks`` repo.

**Comment by jspaaks** [comment]
The contents of this issue have been resolved. Closing.

**Pull request closed without merging**